### PR TITLE
Quiz api

### PR DESCRIPTION
**Pull request by ahohnsen**

### CODE REVIEW

File: components/Card.js [hunk 1]
```diff
@@ -19,4 +19,15 @@ export default function Card(cardElement) {
     }
     answerText.classList.toggle('card__answer--hide');
   });
+
+  const apiUrl = 'https://jservice.io/api/random';
```
Comment (Quaquaro) [inline components/Card.js:23]: Maybe you would like to work with the OPENTRIVIA DB, where you will find real quiz questions.

File: components/Card.js [hunk 2]
```diff
@@ -19,4 +19,15 @@ export default function Card(cardElement) {
...
+  fetch(apiUrl)
+    .then(response => response.json())
+    .then(data => createQuizCard(data))
+    .catch(error => console.log(error));
+
+  function createQuizCard(card) {
+    questionText.innerHTML = `<p>${card[0].question}</p>`;
```
Comment (Quaquaro) [inline components/Card.js:30]: I also want to use the procedure via the innerHTML method. At the moment I am still using textContent.

**Review by Quaquaro — COMMENTED** [review]
The code is clear and well structured.

File: components/Card.js (hunk 1 above)
Comment (ahohnsen) [reply components/Card.js:23]: Thanks for the hint :)

File: components/Card.js (hunk 2 above)
Comment (ahohnsen) [reply components/Card.js:30]: textContent did not work for me, cause I am using a paragraph for the questions & answers

**Pull request merged**